client/lib/plans/constants.js

```diff
@@ -137,9 +140,242 @@ export const FEATURE_CONCIERGE_SETUP = 'concierge-setup-jetpack';
 export const FEATURE_MARKETING_AUTOMATION = 'marketing-automation';
 export const FEATURE_SEARCH = 'search';
 
+// Meta grouping constants
+export const GROUP_WPCOM = 'GROUP_WPCOM';
+export const GROUP_JETPACK = 'GROUP_JETPACK';
+
+export const TERM_MONTHLY = 'TERM_MONTHLY';
+export const TERM_ANNUALLY = 'TERM_ANNUALLY';
+export const TERM_BIENNIALLY = 'TERM_BIENNIALLY';
+
+export const TYPE_FREE = 'TYPE_FREE';
+export const TYPE_PERSONAL = 'TYPE_PERSONAL';
+export const TYPE_PREMIUM = 'TYPE_PREMIUM';
+export const TYPE_BUSINESS = 'TYPE_BUSINESS';
+
+const getPlanPersonalDetails = () => ( {
+	group: GROUP_WPCOM,
+	type: TYPE_PERSONAL,
+	getTitle: () => i18n.translate( 'Personal' ),
+	getAudience: () => i18n.translate( 'Best for hobbyists' ),
+	getBlogAudience: () => i18n.translate( 'Best for hobbyists' ),
+	getPortfolioAudience: () => i18n.translate( 'Best for hobbyists' ),
+	getStoreAudience: () => i18n.translate( 'Best for hobbyists' ),
+	getDescription: () =>
+		i18n.translate(
+			'{{strong}}Best for Personal Use:{{/strong}} Boost your' +
```

Comment:
Hi! I've found a possible matching string that has already been translated **35** times:
`translate( '{{strong}}Best for Personal Use:{{/strong}} Boost your website with a custom domain name, and remove all WordPress.com advertising. Get access to high-quality email and live chat support.' )` *ES Score: 17*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).